content/java/concepts/stringbuilder/stringbuilder.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'StringBuilder'
+Description: 'The StringBuilder class in Java represents a String-like character sequence that is mutable.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`StringBuilder`** class in Java represents a [`String`](https://www.codecademy.com/resources/docs/java/strings)-like character sequence that is mutable. This provides an alternative to the `String` class when it's a requirement to change a character sequence once it is defined.
+
+It is similar to another class, `StringBuffer`, but it is faster in most circumstances. The difference between `StringBuilder` and `StringBuffer` is that `StringBuilder` is not thread-safe. Because `StringBuilder` offers no guarantee of synchronization, it should not be used across multiple [threads](https://www.codecademy.com/resources/docs/java/threading).
```

Comment:
Consider the following sentence instead: "`StringBuilder` offers no guarantee of synchronization so it should not be used across multiple"